conf/messages.en

```diff
@@ -743,3 +743,7 @@ importCustomsOfficeCode.error.required = Enter the customs office code
 importCustomsOfficeCode.error.length = Customs office code must be 8 characters
 importCustomsOfficeCode.error.invalidCharacter = Customs office code must not contain < and > and : and ;
 importCustomsOfficeCode.error.customsOfficeCodeRegex = Customs office code starts with 2 letters, followed by 6 numbers or 6 mixed letters and numbers
+importCustomsOfficeCode.checkYourAnswers.label = Customs office code
```

Comment:
`importCustomsOfficeCode.change.hidden` is missing from from `en` and `cy` messages
[Optional] consider adding test to the view spec